src/sentry/api/serializers/models/organization.py

```diff
@@ -203,11 +235,49 @@ def serialize(self, obj, attrs, user):
         }
 
 
+class _DetailedOrganizationSerializerResponseOptional(OrganizationSerializerResponse, total=False):
+    role: Any  # TODO replace with enum/literal
+
+
+class DetailedOrganizationSerializerResponse(_DetailedOrganizationSerializerResponseOptional):
+    experiments: Any
+    quota: Any
+    isDefault: bool
+    defaultRole: bool
+    availableRoles: List[Any]  # TODO replace with enum/literal
+    openMembership: bool
+    allowSharedIssues: bool
+    enahncedPrivacy: bool
+    dataScrubber: bool
+    dataScrubberDefaults: bool
+    sensitiveFields: List[Any]  # TODO
+    safeFields: List[Any]
+    storeCrashReports: Any  # TODO
+    attachmentsRole: Any  # TODO
+    debugFilesRole: str
+    eventsMemberAdmin: bool
+    alertsMemberWrite: bool
+    scrubIPAddresses: bool
+    scrapeJavaScript: bool
+    allowJoinRequests: bool
+    relayPiiConfig: str
+    apdexThreshold: int
+    trustedRelays: Any  # TODO
+    access: FrozenSet[str]
+    pendingAccessRequests: int
+    onboardingTasks: Any  # TODO
+
+
 class DetailedOrganizationSerializer(OrganizationSerializer):
-    def get_attrs(self, item_list, user, **kwargs):
+    def get_attrs(
+        self, item_list: Sequence[Organization], user: User, **kwargs: Any
+    ) -> MutableMapping[Organization, MutableMapping[str, Any]]:
         return super().get_attrs(item_list, user)
 
-    def serialize(self, obj, attrs, user, access):
+    def serialize(  # type: ignore
```

Comment:
Why is this type: ignore?

Reply:
mypy isn't happy that this `serialize` function takes an access object and the parent doesn't (Liskov substitution yada yada). 

I didn't want to make any refactors along with these changes to limit scope. I can add a TODO. lmk if that sounds good

Reply:
Ah I see. OK makes sense then. I think a comment/TODO is a good idea so people don't ask again.